test/unit/asset-fs-extra/input.js

```diff
@@ -0,0 +1,3 @@
+const fs = require("fs-extra");
```

Comment:
Hmmm interesting. I thought `fs` would work based on my previous empirical knowledge. Did you just replace `fs-extra` with `fs` in this test? `fs.readJsonSync` only exists in `fs-extra`, so we need to remove this line too.

Reply:
Yep! Would you like to update this PR with the fix as well?

Reply:
It might take me a bit of time to figure things out, but happy to do this with your help! If the fix is trivial, feel free to push it right away or recycle the diff from my test.

Do we want to also include a fix for `graceful-fs`? Or would that be out of scope? Just saw it in project deps and it [looks quite popular](https://www.npmtrends.com/graceful-fs).

Reply:
Yeah probably need both `fs-extra` and `graceful-fs`. I don't know the fix off the top of my head but I'm guessing its as simple as finding the current static analysis for `fs` and also adding a check for `fs-extra` and `graceful-fs`.

Reply:
I think I figured it out. Could you please trigger the CI run?

Reply:
You should be able to run locally with `yarn build` and `yarn test test/unit.test.js`

Reply:
Yep, new tests work locally.